genesis/genesis.go

```diff
@@ -110,6 +113,7 @@ func FromConfig(config *Config) ([]byte, ids.ID, error) {
 				Amount:  json.Uint64(5 * units.MegaAvax),
 			},
 		)
+		initialSupply += 5 * units.MegaAvax
```

Comment:
Should we make these constants?

Reply:
These values are going to get moved into the config to support the real genesis. So I'll address this in a future PR.